docs/sources/shared/back-up/back-up-grafana.md

```diff
@@ -17,8 +17,10 @@ Copy Grafana configuration files that you might have modified in your Grafana de
 
 The Grafana configuration files are located in the following directories:
 
-- Default configuration: `$WORKING_DIR/conf/defaults.ini`
-- Custom configuration: `$WORKING_DIR/conf/custom.ini`
+- Default configuration: `$WORKING_DIR/conf/defaults.ini` (Don't change this file)
+- Custom configuration: `$WORKING_DIR/conf/custom.ini` or `$WORKING_DIR/grafana/grafana.ini`
```

Comment:
I am not sure if we should refer `$WORKING_DIR/grafana/grafana.ini` as `$WORKING_DIR/grafana.ini`